### ISSUE TITLE
Automated publishing of Meshery core definitions

### ISSUE THREAD

**Issue by leecalcote** [meshery/schemas#60]
Is there a conjoining PR in meshery/meshery to delete these definitions?

Is the registry model generator configured to source components from `meshery/schemas`? 

Or is there workflow that needs to populate meshery/meshery with updates to these schemas? // @MUzairS15

_Originally posted by @leecalcote in https://github.com/meshery/schemas/issues/58#issuecomment-1998911144_

**Comment by leecalcote** [comment]
Is this needed? https://github.com/meshery/schemas/pull/40 // @yash37158

**Comment by leecalcote** [comment]
@Yashsharma1911 @MUzairS15 has this been implemented?

**Comment by leecalcote** [comment]
If v1beta1 of components is ready to release, new definitions in https://github.com/meshery/schemas/issues/73 will have already been created and pushed, correct?

**Comment by MUzairS15** [comment]
Yes we are referencing defs in schemas repository and automatically sync those to meshery repo.

Release of `meshery` will also necessitate release of `schemas` with new v1beta1 defs. Those will be updated.

**Comment by leecalcote** [comment]
Thanks for confirming. I tend to think of the direction of our development differently. I think that being schema to definition to declaration to instance is the way that our development should go.

**Comment by MUzairS15** [comment]
Yes I agree with the order of development. Because for core models are curated by us, we need to define it somewhere, instead of skipping the first step (starting from defs) we can start from the schema.
